main.go

```diff
@@ -32,48 +96,317 @@ func main() {
 		log.Fatal("failed to connect to database:", err)
 	}
 
-	rows, err := db.QueryContext(ctx, "SELECT * FROM customers")
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer rows.Close()
+	// rows, err := db.QueryContext(ctx, "SELECT * FROM customers")
+	// if err != nil {
+	// 	log.Fatal(err)
+	// }
+	// defer rows.Close()
 
-	for rows.Next() {
-		var id int
-		var email string 
-		var state string
+	// for rows.Next() {
+	// 	var id int
+	// 	var email string
+	// 	var state string
 
-		if err := rows.Scan(&id, &email, &state); err != nil {
-			log.Fatal(err)
-		}
+	// 	if err := rows.Scan(&id, &email, &state); err != nil {
+	// 		log.Fatal(err)
+	// 	}
 
-		fmt.Println(id, email, state)
-	}
+	// 	fmt.Println(id, email, state)
+	// }
 
-	if err := rows.Err(); err != nil {
-		log.Fatal(err)
-	}
+	// if err := rows.Err(); err != nil {
+	// 	log.Fatal(err)
+	// }
 
 	app := &cli.App{
 		Name: "store",
 		Commands: []*cli.Command{
 			{
-				Name:  "example",
-				Usage: "show a bunch of stuff",
+				Name:  "create-customer",
+				Usage: "Creates a new customer to go in the customers database, must specify email and state(2 letter code)",
+				Action: func(cCtx *cli.Context) error {
+					if cCtx.NArg() < 2 {
+						return errors.New("Must specify email and state")
+					}
+					email := cCtx.Args().Get(0)
+					state := cCtx.Args().Get(1)
+					if len(state) != 2 {
+						return errors.New("State length must be 2")
+					}
+					_, ok := states[strings.ToUpper(state)]
+					if !ok {
+						return errors.New("State must be a valid U.S. State or Territory")
+					}
+
+					insertStatement := "INSERT INTO Customers (email, state) VALUES (?, ?)"
+					res, err := db.Exec(insertStatement, email, state)
+
+					if err != nil {
+						return err
+					}
+					customerID, err := res.LastInsertId()
+					if err != nil {
+						return err
+					}
+					w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', tabwriter.AlignRight|tabwriter.Debug)
+					fmt.Fprintln(w, "Customer_id\tEmail\tState\t")
+					fmt.Fprintf(w, "%d\t%s\t%s\t\n", customerID, email, state)
+					w.Flush()
```

Comment:
So this printing logic is basically the same as the printing logic we see in the customersPrintHelper. 
Can we make this code more DRY (DRY = "don't repeat yourself") by placing any/all logic around printing a customer into a single function? (hint: we can use [structs](https://gobyexample.com/structs) to help). 

If structs and/or object-oriented design are new concepts, happy to dive into those.